Dockerfile

```diff
@@ -1,54 +1,31 @@
-FROM ruby:2.6.10-slim as base
+FROM ruby:2.6.10-slim
 
-# upgrade
```

Comment:
Because it crashes the travis-yml service once you deploy it and this is what I see in the pod logs

```
bundler: command not found: puma
Install missing gem executables with `bundle install`
```

After having a discussion with @GbArc we concluded to rollback the changes. Do you think rollback the changes breaks anything?